styles.css

```diff
@@ -17,5 +22,54 @@
 }
 
 /** Project styling */
+html {
+  font-family: sans-serif;
+}
+
+body {
+  width: 50%;
+  max-width: 800px;
+  min-width: 480px;
+  margin: 0 auto;
+}
+
+.center{
+  text-align: center;
+}
+
+.btnCenter{
+  display: block;
+  margin: 10px auto;
+  text-align: center;
+}
+
+.lightBlue{
+  color: lightblue;
+}
+
+.arrowStyleBullet{
+  list-style-type: "→ ";
+  padding: 0;
+}
+
+section{
+  margin: 20px;
+  display: flex;
+  width: 100%;
+  gap: 16px;
+  align-items: center;
+}
+
+.simple-border{
+  border: 1px solid green;
```

Comment:
I wish you'd worked on your own styling -- your page could have been more effective with just a little more effort and not just copy/pasting a stylesheet we had in class before